design/design-doc-stacks.md

```diff
@@ -257,23 +268,110 @@ owners:
 # Human readable company name.
 company: Upbound
 
+# Category that this Stack best fits
+category: Database
+
 # Keywords that describe this application and help search indexing
 keywords:
 - "samples"
 - "examples"
 - "tutorials"
 
 # Links to more information about the application (about page, source code, etc.)
-links:
-- description: Website
-  url: "https://upbound.io"
-- description: Source Code
-  url: "https://github.com/crossplaneio/sample-extension"
+website: "https://upbound.io"
+source: "https://github.com/crossplaneio/sample-extension"
 
 # License SPDX name: https://spdx.org/licenses/
 license: Apache-2.0
 ```
 
+### Example `install.yaml`
+
+The `install.yaml` file is expected to conform to a standard Kubernetes YAML file describing a single `Deployment` object.
+
+```yaml
```

Comment:
This may be solved by whatever install process is eventually landed on, but just an observation that it seems most stacks will have a very similar `install.yaml`, so there may be an opportunity to generate this either via a stacks CLI or in the stack manager itself. Likely out of scope for this PR.

Reply:
https://github.com/crossplaneio/crossplane/issues/614 and https://github.com/crossplaneio/crossplane/issues/503 should get into this.

Additionally, we have open issues (or perhaps questions) about adding support for multiple deployments from a single install.yaml.  We've already merged support for Jobs via install.yaml, and DaemonSet may needed too.  Hopefully our `install.yaml` files get more diverse soon.